examples/tsp_a280.rs

```diff
@@ -0,0 +1,484 @@
+use petgraph::prelude::*;
```

Comment:
You probably didn't mean to include this in the PR? or am I mistaken?

Reply:
You mean the full example right? It's your call. I'd think it be nice to have a non-trivial example with actual costs etc, including at some later point a cut callback for tightening the LP relaxation. This way it would be more obvious that the interaction of all these different callbacks works.

Reply:
I agree, but I would instead modify the tsp example that's there instead of adding a new one.